components/Header.js

```diff
@@ -1,9 +1,19 @@
+import Image from 'next/image';
 import styled from 'styled-components';
 
-const Title = styled.h1`
-  font-size: 40px;
-  color: ${({ theme }) => theme.colors.primary};
-  margin: 40px 0;
+import logo from '../public/assets/images/logo.png';
+
+const StyledHeader = styled.div`
```

Comment:
```suggestion
const StyledHeader = styled.header`
```

since it's a header, let's use the header element for a better semantic.